spec/models/provider_spec.rb

```diff
@@ -347,6 +347,19 @@
     end
   end
 
+  describe "training_providers" do
+    let(:accredited_provider) { create(:provider, accrediting_provider: "Y") }
```

Comment:
There's a trait `accredited_body` we can use to set this to `Y`. 😭

Reply:
Fixed in 3c2e7a0